src/AggregateFunctions/DDSketchEncoding.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <vector>
+#include <stdexcept>
+
+namespace DB
+{
+class DDSketchEncoding
+{
+public:
+    static constexpr UInt8 numBitsForType = 2;
+    static constexpr UInt8 flagTypeMask = (1 << numBitsForType) - 1;
+    static constexpr UInt8 subFlagMask = ~flagTypeMask;
+
+    class Flag
+    {
+    public:
+        UInt8 byte;
+        Flag(UInt8 t, UInt8 s) : byte(t | s) { }
+        UInt8 Type() const { return byte & flagTypeMask; }
+        UInt8 SubFlag() const { return byte & subFlagMask; }
```

Comment:
This function doesn't seem to be used anywhere. You could remove it or perhaps do:
```suggestion
        [[maybe_unused]] UInt8 SubFlag() const { return byte & subFlagMask; }
```